protobuf/answer.proto

```diff
@@ -26,9 +26,8 @@ package grakn.protocol;
 
 message Answer {
     oneof answer {
-        AnswerGroup answer_group = 1;
-        ConceptMap concept_map = 2;
-        Number number = 3;
+        ConceptMap concept_map = 1;
+        Number number = 2;
```

Comment:
I wonder if we should pick a less ambiguous term here - `NumericAnswer numeric = 2`, perhaps?

Reply:
We decided not to.